napari/utils/events/event.py

```diff
@@ -1050,12 +1050,14 @@ def add(
 
             if inspect.isclass(emitter) and issubclass(emitter, Event):  # type: ignore
                 emitter = EventEmitter(
-                    source=self.source, type_name=name, event_class=emitter  # type: ignore
+                    source=self.source,
+                    type_name=name,
+                    event_class=emitter,  # type: ignore
```

Comment:
Intended deviation (https://docs.astral.sh/ruff/formatter/black/#pragma-comments-are-ignored-when-computing-line-width), so we'd need to ensure the `# type: ignore` comment is on the right lines.